tools/who_what_benchmark/requirements.txt

```diff
@@ -9,3 +9,4 @@ pandas>=2.0.3
 numpy>=1.23.5
 tqdm>=4.66.1
 diffusers
+datasets<3.2.0
```

Comment:
can you  report this issue to https://github.com/huggingface/datasets/issues ?

Reply:
https://github.com/huggingface/datasets/issues/7357